samples/apps/copilot-chat-app/webapp/src/redux/features/plugins/pluginsSlice.ts

```diff
@@ -18,11 +18,11 @@ export const pluginsState = createSlice({
                     break;
                 case Plugins.Jira:
                     plugin = state.Jira;
-
-                    // TODO: Aman to change with Jira integration
-                    const encodedData = Buffer.from(
+                    
+                    const encodedData = 
+                    Buffer.from(
                         `${action.payload.username}:${action.payload.accessToken}`,
-                    ).toString('base64');
+                    ).toString();
                     authData = encodedData;
```

Comment:
you can simplify this down to  authData =`${action.payload.username}:${action.payload.accessToken}` and get rid of lines 21-25 since it's not encoded anymore